bin/MadGraph5_aMCatNLO/cards/production/14TeV/st_tch_4f_ckm_NLO_antitop_powheg_madspin_leptonic/st_tch_4f_ckm_NLO_antitop_powheg_madspin_proc_card.dat

```diff
@@ -0,0 +1,4 @@
+import model loop_sm-ckm
+generate p p > t~ b j $$ w+ w- [QCD] @0
```

Comment:
since I never used the powheg+madspin workflow, I have a naive question: do you also need the proc_card?!?